django/django-quiz.md

```diff
@@ -1,333 +1,333 @@
+#### Q4. A client wants their site to be able to load "Rick & Morty" episodes by number or by titleÃƒÆ’Ã†â€™Ãƒâ€šÃ‚Â¯ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â¿ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â½e.g., shows/3/3 or shows/picklerick. Which URL pattern do you recommend?
 
-    - url(r'shows/<int:season>/<int:episode>/', views.episode_number)
-    👍 - path('shows/<int:season>/<int:episode>/', views.episode_number),
-    - path('shows/<int:season>/<int:episode>', views.episode_number),
-    - url(r'^show/(?P<season>[0-9]+)/(?P<episode>[0-9]+)/$', views.episode_number),
+    - [] url(r'shows/<int:season>/<int:episode>/', views.episode_number)
+    - [x] - path('shows/<int:season>/<int:episode>/', views.episode_number),
+    - [] path('shows/<int:season>/<int:episode>', views.episode_number),
+    - [] url(r'^show/(?P<season>[0-9]+)/(?P<episode>[0-9]+)/$', views.episode_number),
 
-Q5. How do you determine at startup time if a piece of middleware should be used?
+#### Q5. How do you determine at startup time if a piece of middleware should be used?
 
-    👍 - Raise MiddlewareNotUsed in the __init__ function of your middleware.
-    - Implement the not_used method in your middleware class.
-    - List the middleware beneath an entry of django.middleware.IgnoredMiddleware.
-    - Write code to remove the middleware from the settings in [app]/__init__.py.
+    - [x] - Raise MiddlewareNotUsed in the __init__ function of your middleware.
+    - [] Implement the not_used method in your middleware class.
+    - [] List the middleware beneath an entry of django.middleware.IgnoredMiddleware.
+    - [] Write code to remove the middleware from the settings in [app]/__init__.py.
 
-Q6. How do you turn off Django’s automatic HTML escaping for part of a web page?
+#### Q6. How do you turn off DjangoÃƒÆ’Ã†â€™Ãƒâ€šÃ‚Â¯ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â¿ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â½s automatic HTML escaping for part of a web page?
 
-    - Place that section between paragraph tags containing the autoescape=off switch.
-    - Wrap that section between { percentage mark autoescape off percentage mark} and {percentage mark endautoescape percentage mark} tags.
-    - Wrap that section between {percentage mark autoescapeoff percentage mark} and {percentage mark endautoescapeoff percentage mark} tags.
-    👍 - You don't need to do anything—autoescaping is off by default.
+    - [] Place that section between paragraph tags containing the autoescape=off switch.
+    - [] Wrap that section between { percentage mark autoescape off percentage mark} and {percentage mark endautoescape percentage mark} tags.
+    - [] Wrap that section between {percentage mark autoescapeoff percentage mark} and {percentage mark endautoescapeoff percentage mark} tags.
+    - [x] - You don't need to do anythingÃƒÆ’Ã†â€™Ãƒâ€šÃ‚Â¯ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â¿ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â½autoescaping is off by default.
```

Comment:
please double-check why there are special characters instead of alphabetical symbols